### ISSUE TITLE
Test DBpedia integration

### ISSUE THREAD

**Issue by brodecva** [odalic/sti#70]
Test that the integration works using at least one suitable provided test case.

**Comment by JanVana** [comment]
I found several problems during the testing, that I would like to fix.

1) There are some version conflicts in the current Maven configuration. Should be easily solved by tracking which project reference conflicting assemblies and adding suitable ignores into poms.

2) Sometimes the search classes generate invalid SPARQL.

3) The dbpedia search uses the homepage parameter from the freebase search.

4) Some input files are not converted and the conversion crashes. Don't know yet the cause of this bug. I have to investigate more.

**Comment by brodecva** [comment]
Point 1) seems too serious to approve the merge request into develop. I will merge the branch only after specific confirmation that this one is sorted out. (Sorry if your already done so.)
